lib/src/commands/create/templates/very_good_core/very_good_core_template.dart

```diff
@@ -19,14 +19,19 @@ class VeryGoodCoreTemplate extends Template {
   Future<void> onGenerateComplete(Logger logger, Directory outputDir) async {
     await installFlutterPackages(logger, outputDir);
     await applyDartFixes(logger, outputDir);
-    _logSummary(logger);
+    _logSummary(logger, outputDir);
   }
 
-  void _logSummary(Logger logger) {
+  void _logSummary(Logger logger, Directory outputDir) {
     logger
       ..info('\n')
       ..created('Created a Very Good App! 🦄')
       ..info('\n')
+      ..info('In order to open your project, type:')
+      ..info('\n')
+      ..info('  \$ cd ${outputDir.path}')
+      ..info('\n')
+      ..info(r'  $ code .')
```

Comment:
I think this `$ code .` assumes the user is using Visual Studio Code. I would personally suggest users to refer to the README.md for "getting started" instructions.

---

Note that Flutter create does not refer to `code .`:

```txt
Creating project hello_world...
Resolving dependencies in hello_world... (1.3s)
Got dependencies in hello_world.
Wrote 129 files.

All done!
You can find general documentation for Flutter at: https://docs.flutter.dev/
Detailed API documentation is available at: https://api.flutter.dev/
If you prefer video documentation, consider: https://www.youtube.com/c/flutterdev

In order to run your application, type:

  $ cd hello_world
  $ flutter run

Your application code is in hello_world/lib/main.dart.
```

Reply:
Yeah I agree with @alestiago here. This assumes an IDE and that they have `code` setup in their path.

Reply:
@alestiago @tomarra So should I suggest users to refer ```Quick Start``` or ```commands``` section in the README.md?
The problem with ```flutter run``` is that when an  app is created using very_good_cli it doesn't generate a main.dart file so its unable to run it.

We could do something like:

Created a Very Good App! 🦄

for more commands to get started, refer:
       <i>the section in readme</i>

Reply:
What about:

```txt
Created a Very Good App! 🦄

To get started refer to app_name/README.md.
      
Your application is located in desktop/app_name.
```

Reply:
Yeah that sounds more clear and straight forward on where my project is as well as what is the next step. Ill update the code accordingly. Thank you for the guidance.

Reply:
@alestiago can you please elaborate difference between Directory.current and Directory.path